tests/testKron.py

```diff
@@ -561,62 +559,50 @@ def testForced(self):
                         shape = source.getShape().clone()
                         xc, yc = source.getCentroid()
                         radius = source.get("ext_photometryKron_KronFlux_radius")
-                        for r, ct in [(radius, ds9.BLUE), (radius*kfac, ds9.CYAN),]:
+                        for r, ct in [(radius, ds9.BLUE), (radius*kfac, ds9.CYAN), ]:
                             shape.scale(r/shape.getDeterminantRadius())
                             ds9.dot(shape, xc, yc, ctype=ct, frame=1)
                         ds9.mtv(warped, frame=2)
-                        transform = (wcs.linearizeSkyToPixel(source.getCoord())*
+                        transform = (wcs.linearizeSkyToPixel(source.getCoord()) *
                                      original.getWcs().linearizePixelToSky(source.getCoord()))
                         shape = shape.transform(transform.getLinear())
                         radius = source.get("ext_photometryKron_KronFlux_radius")
                         xc, yc = wcs.skyToPixel(source.getCoord()) - afwGeom.Extent2D(warped.getXY0())
-                        for r, ct in [(radius, ds9.BLUE), (radius*kfac, ds9.CYAN),]:
+                        for r, ct in [(radius, ds9.BLUE), (radius*kfac, ds9.CYAN), ]:
                             shape.scale(r/shape.getDeterminantRadius())
                             ds9.dot(shape, xc, yc, ctype=ct, frame=2)
                     try:
                         self.assertClose(source.get("ext_photometryKron_KronFlux_flux"),
```

Comment:
I feel like we should instead do a "deprecation pass", running all our tests with "-W all", since there are certainly other deprecations we should take care of.